src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Redis/StackExchange/RedisExecuteAsyncIntegration.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ClrProfiler.Integrations;
+using Datadog.Trace.ClrProfiler.Integrations.StackExchange.Redis;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange
+{
+    /// <summary>
+    /// StackExchange.Redis.[RedisBase/RedisBatch/RedisTransaction].ExecuteAsync[T] calltarget instrumentation
+    /// </summary>
+    [RedisExecuteAsyncInstrumentMethod(TypeName = "StackExchange.Redis.RedisBase")]
+    [RedisExecuteAsyncInstrumentMethod(TypeName = "StackExchange.Redis.RedisBatch")]
+    [RedisExecuteAsyncInstrumentMethod(TypeName = "StackExchange.Redis.RedisTransaction")]
```

Comment:
Looks like we only instrument RedisBatch: https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/RedisBatch.cs#L127

Reply:
`RedisBase` must be instrumented because there is code used by `RedisBatch` that is implemented in the parent class. I added `RedisTransaction` because also inherits `RedisBase`, but I can remove it if we need to.

In the case `RedisTransaction` we are missing the `ExecuteAsync` doesn't call the base class: https://github.com/StackExchange/StackExchange.Redis/blob/59ef0c4eb4bed27014521e0bf0c24bd18fcf596e/src/StackExchange.Redis/RedisTransaction.cs#L62-L109